chapters/lit_review/angle_VR.tex

```diff
@@ -282,6 +282,59 @@ \subsubsection{Angular Biasing Using the Exponential Transform}
 evenly distributing the problems in problems with anisotropy.
 MCNP Weight Window Generator \textbf{cite MCNP manual here}
 
+
+The AVATAR method
```

Comment:
may want to change to present rather than past tense. The method still exists, so it doesn't completely make sense to talk about it as past tense.